book/forms.rst

```diff
@@ -699,8 +699,8 @@ the documentation for each type.
     is left blank. If you don't want this behavior, either
     :ref:`disable HTML5 validation <book-forms-html5-validation-disable>`
     or set the ``required`` option on your field to ``false``::
-    
-        ->add('dueDate', DateType::class, array(
+
+        ->add('dueDate', 'date', array(
```

Comment:
This should have been `Symfony\Component\Form\Extension\Core\Type\Date`, shouldn't it?